packages/slidev/node/export.ts

```diff
@@ -70,13 +69,17 @@ export async function exportSlides({
   timeout = 500,
   dark = false,
   routerMode = 'history',
-  width = 1920,
-  height = 1080,
+  width = 980,
+  height = 551,
```

Comment:
What's the reason of this change?

Reply:
Because the slides size are 980/551 by default (from `canvasWidth` and `aspectRatio`).
There is no need to open a browser that is 1920 wide because we only use 980px width maximum.
I can change the numbers to use the config values instead.

Reply:
So I can revert default values in fact, that's not a problem if you prefer so.